tests/helper/helper_dev.go

```diff
@@ -82,17 +84,32 @@ import (
 		})
 	})
 */
+
+const LOCALHOST_REGEXP = "localhost:[0-9]*"
```

Comment:
```suggestion
const localhostRegexp = "localhost:[0-9]*"
```

This should not be exported